internal/cel/library/kubernetes_test.go

```diff
@@ -98,3 +99,49 @@ func TestKubernetes(t *testing.T) {
 		})
 	}
 }
+
+func TestKubernetesHostFailure(t *testing.T) {
+	tests := []struct {
+		name        string
+		expression  string
+		errorString string
+	}{
+		{
+			"kw.k8s.listAllResources host failure",
+			"kw.k8s.listAllResources(ListAllResourcesRequest{Kind: 'Pod'})",
+			"cannot list all Kubernetes resources: hostcallback error",
+		},
+		{
+			"kw.k8s.listResourcesByNamespace",
+			"kw.k8s.listResourcesByNamespace(ListResourcesByNamespaceRequest{Namespace: 'default'})",
+			"cannot list Kubernetes resources by namespace: hostcallback error",
+		},
+		{
+			"kw.k8s.getResource",
```

Comment:
Just in case you want to keep the pattern from the previous test name. This is not a blocker to merge this PR
```suggestion
			"kw.k8s.getResource host failure",
```